core/modules/user/user.theme.inc

```diff
@@ -386,3 +386,53 @@ function template_preprocess_user_profile(&$variables) {
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid;
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid . '__' . $view_mode;
 }
+
+/**
+ * Preprocess variables for user-simplified-page.tpl.php.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: The form structure to be rendered.
+ *   - links: A render array containing the links to other related forms.
+ *
+ * @see user-simplified-page.tpl.php
+ */
+function template_preprocess_user_simplified_page(&$variables) {
+  $page_title = backdrop_get_title();
+  $variables['logo_image'] = '';
+
+  if ($logo = backdrop_get_logo()) {
+    $variables['logo_image'] = theme('image', array(
+      'path' => $logo,
+      'alt' => $page_title,
+    ));
+  }
+  $variables['site_name'] = config_get_translated('system.core', 'site_name');
+  $variables['front_page'] = url();
+  $variables['page_title'] = $page_title;
+  $variables['messages'] = theme('status_messages');
+  backdrop_add_css(backdrop_get_path('module', 'user') . '/css/user-simplified-page.css');
+}
+
+/**
+ * Returns HTML for user forms with links instead of tabs.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: A render array of the form to be rendered
+ *     (e.g. login or password reset).
+ *   - links: An item_list render array with the local tasks as links. 
```

Comment:
This line contains trailing white space. :wink: 

And the above line contains the (seemingly problematic) "e.g.".